.github/workflows/dev-ci-cd.yml

```
@@ -0,0 +1,96 @@
+name: Development CI/CD
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)